packages/relay-compiler/language/RelayLanguagePluginInterface.js

```diff
@@ -34,6 +49,9 @@ export type PluginInterface = {
   findGraphQLTags: GraphQLTagFinder,
   formatModule: FormatModule,
   typeGenerator: TypeGenerator,
+  processCompilerTransforms?: CompilerTransforms => CompilerTransforms,
+  extraSchema?: string,
```

Comment:
Let’s rename this to `schemaAdditions` and add some doc with an example of adding language specific directives.

Reply:
Or perhaps `schemaExtensions`, as per the `extendSchema` API 🤔